scripts/scil_compute_msmt_frf.py

```diff
@@ -0,0 +1,250 @@
+             'used for computations and reconstruction. Useful if no tissue '
+             'masks are available.')
+    p.add_argument(
+        '--mask_wm',
+        help='Path to the input WM mask file, used to improve the final WM '
+             'frf mask.')
+    p.add_argument(
+        '--mask_gm',
+        help='Path to the input GM mask file, used to improve the final GM '
+             'frf mask.')
+    p.add_argument(
+        '--mask_csf',
+        help='Path to the input CSF mask file, used to improve the final CSF '
+             'frf mask.')
+
+    p.add_argument(
+        '--fa_thr_wm', default=0.7, type=float,
+        help='If supplied, use this threshold to select single WM fiber '
+             'voxels from the FA inside the WM mask defined by mask_wm. Each '
+             'voxel above this threshold will be selected. [%(default)s]')
+    p.add_argument(
+        '--fa_thr_gm', default=0.2, type=float,
+        help='If supplied, use this threshold to select GM voxels from the FA '
+             'inside the GM mask defined by mask_gm. Each voxel below this '
+             'threshold will be selected. [%(default)s]')
+    p.add_argument(
+        '--fa_thr_csf', default=0.1, type=float,
+        help='If supplied, use this threshold to select CSF voxels from the '
+             'FA inside the CSF mask defined by mask_csf. Each voxel below '
+             'this threshold will be selected. [%(default)s]')
+    p.add_argument(
+        '--md_thr_gm', default=0.0007, type=float,
+        help='If supplied, use this threshold to select GM voxels from the MD '
+             'inside the GM mask defined by mask_gm. Each voxel below this '
+             'threshold will be selected. [%(default)s]')
+    p.add_argument(
+        '--md_thr_csf', default=0.003, type=float,
+        help='If supplied, use this threshold to select CSF voxels from the '
+             'MD inside the CSF mask defined by mask_csf. Each voxel below '
+             'this threshold will be selected. [%(default)s]')
+
+    p.add_argument(
+        '--min_nvox', default=100, type=int,
+        help='Minimal number of voxels needed for each tissue masks '
+             'in order to proceed to frf estimation. [%(default)s]')
+    p.add_argument(
+        '--tolerance', type=int, default=20,
+        help='The tolerated gap between the b-values to '
+             'extract\nand the current b-value. [%(default)s]')
+    p.add_argument(
+        '--dti_bval_limit', type=int, default=1200,
+        help='The highest b-value taken for DTI. [%(default)s]')
+    p.add_argument(
+        '--roi_radii', default=[10], nargs='+', type=int,
+        help='If supplied, use those radii to select a cuboid roi '
+             'to estimate the response functions. The roi will be '
+             'a cuboid spanning from the middle of the volume in '
+             'each direction with the different radii. The type is '
+             'either an int or an array-like (3,). [%(default)s]')
```

Comment:
How to provide the radii is not clear, especially with the default printed as [[10]]. Versus --roi_center below with the tuple(3).

Can you please try to clarify both, and, here an example would be useful, either at the end of the initial help of the text or here.

For instance, my command line (which took me 10 minutes to figure out...) looks like this
scil_compute_msmt_frf.py ../../HCP/7T/100610/Resample_DWI/100610__dwi_resampled.nii.gz ../../HCP/7T/100610/Eddy/100610__bval_eddy ../../HCP/7T/100610/Eddy/100610__dwi_eddy_corrected.bvec wm_frf.txt gm_frf.txt csf_frf.txt --mask_wm ../../HCP/7T/100610/Segment_Tissues/100610__mask_wm.nii.gz --mask_gm ../../HCP/7T/100610/Segment_Tissues/100610__mask_gm.nii.gz --mask_csf ../../HCP/7T/100610/Segment_Tissues/100610__mask_csf.nii.gz --tolerance 200 --force_b0_threshold --frf_table frf_table.txt --wm_frf_mask frf_mask_wm.nii.gz --gm_frf_mask frf_mask_gm.nii.gz --csf_frf_mask frf_mask_csf.nii.gz -v --md_thr_gm 0.001 --fa_thr_gm 0.5 --roi_center 66 79 79 --roi_radii 20 30 10 -f

It would have helped me on first run to see an example...